src/routes/safe/components/Apps/components/AppsList.test.tsx

```diff
@@ -5,6 +5,7 @@ import { render, screen, fireEvent, within, act, waitFor } from 'src/utils/test-
 import * as appUtils from 'src/routes/safe/components/Apps/utils'
 import { FETCH_STATUS } from 'src/utils/requests'
 import { loadFromStorage, saveToStorage } from 'src/utils/storage'
+import { CURRENT_SESSION_REDUCER_ID } from 'src/logic/currentSession/store/reducer/currentSession'
 
 jest.mock('src/routes/routes', () => {
```

Comment:
I think this mock can be removed now that `extractSafeAddress` doesn't exist anymore